core-kotlin-modules/core-kotlin-7/src/main/kotlin/com/baeldung/optional/Optional.kt

```diff
@@ -0,0 +1,33 @@
+package com.baeldung.optional
+
+import java.util.Optional
+import kotlin.jvm.optionals.getOrDefault
+import kotlin.jvm.optionals.getOrElse
+import kotlin.jvm.optionals.getOrNull
+
+fun getOrNull() {
+    val optionalValue: Optional<String> = Optional.of("Hello, Baeldung!")
+    val value: String? = optionalValue.getOrNull()
+}
+
+fun getOrElse() {
+    val optionalValue: Optional<String> = Optional.empty()
+    val value: String = optionalValue.getOrElse { throw IllegalStateException("Value is missing") }
+    val value2: String = optionalValue.getOrElse { "Default Value" }
+}
+
+fun getOrDefault() {
+    val optionalValue: Optional<String> = Optional.empty()
+    val value1: String = optionalValue.getOrDefault("Default Value")
+}
```

Comment:
Let's also actually delete these 3 functions, since the test is a better demonstration of usage and so that we don't duplicate the code

Reply:
```suggestion
```